### PR TITLE
Update the connected registry deployment instruction on nest iot.

### PR DESCRIPTION
**Pull request by huanwu**

### CODE REVIEW

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md [hunk 1]
```diff
@@ -163,12 +161,76 @@ To deploy the connected registry module using the Azure CLI, save the following
...
+## Setup and deploy the connected registry module on nested IoT Edge
+This tutorial requires a nested Azure IoT Edge device to be set up upfront. You can use the [Deploy your first IoT Edge module to a virtual Linux device](https://docs.microsoft.com/en-us/azure/iot-edge/quickstart-linux?view=iotedge-2020-11) quickstart guide to learn how to deploy a virtual IoT Edge device. To create a nested IoT Edge devices, follow the instructions [Tutorial: Create a hierarchy of IoT Edge devices](https://docs.microsoft.com/en-us/azure/iot-edge/tutorial-nested-iot-edge?view=iotedge-2020-11) to learn how to configure hierarchical IoT edge devices. The connected registry is deployed as a module on the nested IoT Edge device.
+
+Based on the tutorial, it overall includes the following steps:
+1. Create top level and lower level vms from existing template. The template will also install the iot agent on it. You can use the [Tutorial: Install or uninstall Azure IoT Edge for Linux](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge?view=iotedge-2020-11) to learn how to manually set up the machine if you need deploy from your own devices.
```
Comment (xyxyxyxyxyxy) [inline docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md:173]: Better to add a reminder here to let customer make sure they open the ports 8000, 443, 5671, 8883 on top level vm

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md [hunk 2]
```diff
@@ -50,21 +36,31 @@ To support nested IoT Edge scenarios, the container image for the connected regi
...
   --name mycontainerregistry001 \
-  --source mcr.microsoft.com/acr/connected-registry:0.2.0
+  --source mcr.microsoft.com/acr/connected-registry:0.3.0
 ```
 
 To learn more about nested IoT Edge scenarios, please visit [Tutorial: Create a hierarchy of IoT Edge devices](https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/iot-edge/tutorial-nested-iot-edge.md).
 
+## Import the IotEdge and API Proxy images into your registry
+
+Import the images azureiotedge-api-proxy, azureiotedge-agent, azureiotedge-hub to your private registry using the same command as above.
```
Comment (northtyphoon) [inline docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md:46]: I see you use the scope-map is using the repository name like  "azureiotedge-api-proxy" "azureiotedge-agent" "azureiotedge-hub"

Can you update the import section to use import command?

az acr import -n mycontainerregistry001 --source acronpremiot.azurecr.io/acr/microsoft/azureiotedge-agent:20210609.5 -t azureiotedge-agent:20210609.5
az acr import -n mycontainerregistry001 --source acronpremiot.azurecr.io/acr/microsoft/azureiotedge-hub:20210609.5 -t azureiotedge-hub:20210609.5
az acr import -n mycontainerregistry001 --source acronpremiot.azurecr.io/acr/microsoft/azureiotedge-api-proxy:9.9.9-dev -t azureiotedge-api-proxy:9.9.9-dev

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md [hunk 3]
```diff
@@ -164,6 +160,27 @@ To deploy the connected registry module using the Azure CLI, save the following
                         "status": "running",
                         "restartPolicy": "always",
                         "version": "1.0"
+                    },
+                    "IoTEdgeAPIProxy": {
+                        "settings": {
+                            "image": "mycontainerregistry001.azurecr.io/acr/microsoft/azureiotedge-api-proxy:9.9.9-dev",
```
Comment (northtyphoon) [inline docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md:166]: mycontainerregistry001.azurecr.io/azureiotedge-api-proxy:9.9.9-dev

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md [hunk 4]
```diff
@@ -183,7 +200,7 @@ To deploy the connected registry module using the Azure CLI, save the following
                 "systemModules": {
                     "edgeAgent": {
                         "settings": {
-                            "image": "mcr.microsoft.com/azureiotedge-agent:1.2",
+                            "image": "mycontainerregistry001.azurecr.io/acr/microsoft/azureiotedge-agent:20210609.5",
```
Comment (northtyphoon) [inline docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md:203]: mycontainerregistry001.azurecr.io/azureiotedge-agent:20210609.5

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md [hunk 5]
```diff
@@ -195,7 +212,7 @@ To deploy the connected registry module using the Azure CLI, save the following
                     },
                     "edgeHub": {
                         "settings": {
-                            "image": "mcr.microsoft.com/azureiotedge-hub:1.2",
+                            "image": "mycontainerregistry001.azurecr.io/acr/microsoft/azureiotedge-hub:20210609.5",
```
Comment (northtyphoon) [inline docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md:215]: mycontainerregistry001.azurecr.io/azureiotedge-hub:20210609.5

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md [hunk 6]
```diff
@@ -83,7 +81,7 @@ To deploy the connected registry module using the Azure CLI, save the following
                     },
                     "IoTEdgeApiProxy": {
                         "settings": {
-                            "image": "10.16.7.4/azureiotedge-api-proxy:latest",
+                            "image": "$upstream/acr/microsoft/azureiotedge-api-proxy:9.9.9-dev",
```
Comment (northtyphoon) [inline docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md:84]: $upstream/azureiotedge-api-proxy:9.9.9-dev

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md [hunk 7]
```diff
@@ -124,7 +122,7 @@ To deploy the connected registry module using the Azure CLI, save the following
                 "systemModules": {
                     "edgeAgent": {
                         "settings": {
-                            "image": "10.16.7.4/azureiotedge-agent:1.2",
+                            "image": "$upstream/acr/microsoft/azureiotedge-agent:20210609.5",
```
Comment (northtyphoon) [inline docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md:125]: $upstream/azureiotedge-agent:20210609.5

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md [hunk 8]
```diff
@@ -136,7 +134,7 @@ To deploy the connected registry module using the Azure CLI, save the following
                     },
                     "edgeHub": {
                         "settings": {
-                            "image": "10.16.7.4/azureiotedge-hub:1.2",
+                            "image": "$upstream/acr/microsoft/azureiotedge-hub:20210609.5",
```
Comment (northtyphoon) [inline docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md:137]: $upstream/azureiotedge-hub:20210609.5

**Review by northtyphoon — CHANGES_REQUESTED** [review]

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md (hunk 3 above)
Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md:166]: The command to import the image is az acr import --name mycontainerregistry001 --source acronpremiot.azurecr.io/acr/microsoft/azureiotedge-api-proxy:9.9.9-dev. Should be mycontainerregistry001.azurecr.io/acr/microsoft/azureiotedge-api-proxy:9.9.9-dev?

Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md:166]: I see you comments on import command and understand what you mean. I will update all of them.

**Review by northtyphoon — APPROVED** [review]

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md (hunk 8 above)
Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md]: Resolved

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md (hunk 7 above)
Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md]: Resolved

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md (hunk 6 above)
Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md]: Resolved

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md (hunk 5 above)
Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md]: Resolved

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md (hunk 4 above)
Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md]: Resolved

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md (hunk 3 above)
Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md]: Resolved

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md (hunk 2 above)
Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md]: Resolved

File: docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md (hunk 1 above)
Comment (huanwu) [reply docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md:173]: Resolved

**Pull request merged**